docs/getting-started.md

```diff
@@ -9,65 +9,234 @@ sidebar_label: Getting started
+If you have problems during the instalation of aragonCLI or any other dependencies. You probably need to install [windows-build-tools](https://www.npmjs.com/package/windows-build-tools) or similar package. 
 
+Again, if you're having trouble fixing things, please reach out to us at the [#dev-help channel on the Aragon Chat](https://aragon.chat/channel/dev-help)
+
+### Note on Git
+
+You might need to have [Git](https://git-scm.com) installed. [a little startling for the reader... why might? i think we need to elaborate here...]
 
 ## Quick start
 
-To create your first DAO run:
+In order to get up and running quickly, we’ll build our first DAO using some basic scaffolding [explain what scaffolding means] -- remember a DAO is just a combination of Aragon Apps... [elaborate...]
+
+To create our DAO run:
 
 ```sh
 npx create-aragon-app foo.aragonpm.eth
 cd foo
 npx aragon run
 ```
 
-Congrats you have just created a DAO! It’s running on your local network and as soon as it's ready it will open in your browser at [localhost:3000](http://localhost:3000)!
+Don't worry about understanding the first line right now. Really. We'll cover that in the [tutorial](https://hack.aragon.org/docs/tutorial.html) coming up.
+
+With respect to the third line, ```aragon run``` just sets up everything for us in the background so we can quickly live test our DAO.
+
+N.B. If you're unsure what the difference is between ```npx``` and ```npm```, we recommend you read through this [stackoverflow post.](https://stackoverflow.com/questions/50605219/difference-between-npx-and-npm)
+
+If you've made it this far congrats 🤗. You've just created your first DAO!
+
+It’s running on your local network and as soon as it's ready it will open in your browser at [localhost:3000](http://localhost:3000)!
+
+### Interacting with your first DAO
+If it's not already open, open your browser at the localhost address shown in your terminal.
+
+![](/docs/assets/getting-started-dao-0.png)
+
+Once your browser is open at the right address, you should see a screen that looks like the one below
+
+![](/docs/assets/getting-started-dao-1.png)
+
+As you can see on the left, this DAO is made up of two Aragon apps -- Home and Counter. Right now, we're in the Home app. This app just displays a welcome message with no possible user interactions.
+
+Click on Counter to open up the (slightly) more interesting Counter app.
+
+![](/docs/assets/getting-started-dao-2.png)
+
+[^ reread and rework this section]
+
+#### Signing your first transaction with Metamask
+
+[insert visual tutorial]
+
+#### Signing your first transaction with Frame
+
+[insert visual tutorial]
 
 ## Next steps
 
-Now that you’ve built a DAO let's take a look at the docs or you can jump into the [tutorial](/docs/tutorial.html).
+We hope you enjoyed that 😊! Please don't hesitate to leave us any feedback [insert link].
```

Comment:
This could be a good place to mention that all the pages across the website documentation have an `Edit` button that link you to the corresponding repo to open a PR to include changes 🙌

Reply:
Done.
Not sure if it exists already, but could be cool to have a mini-tutorial that takes the user through their first Aragon github edit. What do you think?

Reply:
Definitely and not sure where is the place for this 🤔

Reply:
How about a Contributing section at the end of the hack docs?

Reply:
I more inclined to include it in the [`Contributing.md`](https://github.com/aragon/hack/blob/master/CONTRIBUTING.md#documentation-guidelines). Maybe make a medium article or a blog post on Aragon blog, to have more reach?

Reply:
I like that. We can link the article in Contributing.md, and link to Contributing.md at the end of the getting-started page. What do you think?

Reply:
Make sense to me👍

Reply:
Started writing the first draft on Medium. I'm assuming no prior experience with either git or github. Should have something to show for this time tomorrow.

Reply:
I think this also solves [this issue.](https://github.com/aragon/aragon-wiki/issues/259)